.gitignore

```diff
@@ -81,6 +81,9 @@ docs/_build/
 # PyBuilder
 target/
 
+# thirdparty dirs
+third_party/*/
```

Comment:
Where is this coming from?

Reply:
This is also in the PR for the regex library where we add `re2` submodule. We also have a build_dir that gets created in the third_party folder. I was thinking git should be ignoring these changes